README.md

```diff
@@ -682,6 +683,19 @@ Example:
 ./pkb.py --cloud=GCP --zone=us-central1-a --benchmarks=iperf,ping
 ```
 
+The disk type names vary by provider, but the following table summarizes some
+useful ones.
+
+Cloud name | Network-attached SSD | Network-attached HDD
+-----------|----------------------|---------------------
+GCP | pd-ssd | pd-standard
+AWS | gp2 | standard
+Azure | premium-storage | standard-disk
+
+Also note that `--data_disk_type=local` tells PKB not to allocate a separate
```

Comment:
When there are 2 disks... remote HDD and local SSD like on some Azure instances how is this resolved?